pipelines/scipipe/ap_verify.groovy

```diff
@@ -290,7 +289,7 @@ def void verifyDataset(Map p) {
               datasetName: ds.name,
               sasquatchUrl: sqre.sasquatch.url,
               branchRefs: codeRef,
-              pipeline: pipeline_path[pipeline_path.length-1]
+              pipeline: Paths.get(ds.gen3_pipeline).getFileName().toString()
```

Comment:
I think you may need an `import java.nio.file.Path` at the top, and then [another file](https://github.com/lsst-dm/jenkins-dm-jobs/blob/0315b05ca227ab947dfc7555b06236e0d16b498c/pipelines/lib/util.groovy#L1895) uses `Path.of()`.

Reply:
It's a bit unfortunate that `Path.of` [has the same weakness](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/nio/file/Path.html#of(java.lang.String,java.lang.String...)) as `Paths.get`, but hopefully it won't matter.